src/snapred/backend/dao/state/CalibrantSample/Atom.py

```diff
@@ -22,6 +22,20 @@ class Atom(BaseModel):
     siteOccupationFactor: float
     adp: float = 0.1
 
+    @validate_arguments
+    def __init__(self, *args: StrictStr, **kwargs):
+        if args:
+            print(args)
```

Comment:
Assuming this print is for testing, probably should remove this

Reply:
Good catch